capstone-backend/src/main/java/com/google/sps/servlets/ClubSearchServlet.java

```diff
@@ -0,0 +1,58 @@
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Map;
+import java.util.concurrent.ExecutionException;
+
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import com.google.cloud.firestore.CollectionReference;
+import com.google.cloud.firestore.DocumentSnapshot;
+import com.google.cloud.firestore.Firestore;
+import com.google.cloud.firestore.Query;
+import com.google.gson.Gson;
+import com.google.sps.data.Club;
+
+@WebServlet("/api/clubSearch")
```

Comment:
The existing 'clubs' servlet actually looks pretty lean at the moment. I think it might make more sense to make this a part of that, or to turn this into a sort of 'sub'-servlet if the framework allows that sort of thing.

Reply:
Initially I considered doing this before since the doGet is currently 4 lines and I feel like all the "club" related calls should go in the same class and not have separate methods.

I'll move my clubSearch code to the clubServlet to make it more simple.